src/runtime/methodwrapper.cs

```diff
@@ -12,24 +12,43 @@ internal class MethodWrapper
     {
         public IntPtr mdef;
         public IntPtr ptr;
+        public ThunkInfo _thunk;
+        private bool _disposed = false;
```

Comment:
Please, don't prefix member names with `_`. If you need to distinguish them, use `this.`

Reply:
`_thunk` should be private, `public` just a mistake. I prefer private members start with `_` on C#, make it easy to distinguish they're public or not. Of course, if the coding standards here is not using any `_`, I'm be willing to remove them.